### PR TITLE
add a the ability to add/delete users dynamically

### LINKED ISSUE #106: Add the ability to add/delete users dynamically
Still in the context of using Chisel as a library, it would be nice to be able to add and remove users dynamically.

### PR DESCRIPTION
**Pull request by deviantony**
Closes #106

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -183,3 +183,14 @@ func (s *Server) authUser(c ssh.ConnMetadata, password []byte) (*ssh.Permissions
 	s.sessions.Set(string(c.SessionID()), user)
 	return nil, nil
 }
+
+// AddUser adds a new user into the server user index
+func (s *Server) AddUser(user, pass, addrs string) {
```
[the remark this answers is not in the document]
Comment (deviantony) [reply server/server.go]: Make sense, I'll push an update soon.

**Comment by deviantony** [comment]
@jpillora done !

**Comment by jpillora** [comment]
Merged :) thanks! Will definitely tag/release this commit tonight

**Comment by deviantony** [comment]
Thanks @jpillora

**Pull request merged**